analyzer/tests/unit/test_checker_handling.py

```diff
@@ -396,6 +399,18 @@ def test_disable_clangsa_checkers(self):
                 analyzer.config_handler.checks()['Wreserved-id-macro'][0],
                 CheckerState.enabled)
 
+    def test_analyze_wrong_parameters(self):
+        """
+        This test checks whether the analyze command detects if a wrong
+        --analyzer-config or --checker-config parameter is specified.
+        """
+
+        analyzer_cfg = [AnalyzerConfig('asd', '123', 'value')]
+        checker_cfg = [CheckerConfig('clangsa', 'asd', '123', 'value')]
+
+        self.assertFalse(is_analyzer_config_valid(analyzer_cfg))
+        self.assertFalse(is_checker_config_valid(checker_cfg))
```

Comment:
Would liketo see some more tests for
* Correct analyzer/checker configs,
* Valid analyzer for --analyzer-config, but invalid config name,
* Valid analyzer for --checker-config and
  * invalid checker name + valid checker option name,
  * valid checker name + invalid checker option name,
* Invalid analyzer for --checker-config.